packages/quantic/force-app/main/default/lwc/quanticCategoryFacet/quanticCategoryFacet.js

```diff
@@ -363,6 +373,10 @@ export default class QuanticCategoryFacet extends LightningElement {
       this.facet.toggleSelect(item);
     }
     this.clearInput();
+    this.focusShouldBeInFacet = true;
```

Comment:
Maybe 1 function to avoir repeating the same code everywhere?

`this.keepFocusInFacet('header'|'value');` ?